### ISSUE TITLE
core dump when calling ht.init() in multiple GPUs mode

### ISSUE THREAD

**Issue by MuYu-zhi** [NVIDIA-Merlin/HugeCTR#243]
I'm trying to build tf model with HugeCTR embedding plugin in hugectr_tf_ops.so. When I call ht.init(), only one GPU mode works well, multiple GPUs mode (2 or 4) fails. My code is as below: 
`
import hugectr_tf_ops as ht
gpus = [0,1,2,3]
ht.init(visiable_gpus=gpus, seed=seed, key_type='int64', value_type='float',
                    batch_size=BATCH_SIZE, batch_size_eval=BATCH_SIZE)
`
The training log blocks at "[06d11h00m33s][HUGECTR][INFO]: Global seed is 1024" and finally ends up with core dump.

**Comment by Jianbing-D** [comment]
Could you please provide a demo about how to reproduce this issue?

**Comment by Jianbing-D** [comment]
Also, we implemented a new embedding plugin, which is called SparseOperationKit and more friendly to original TensorFlow, and encourage users to try that python module. The documents for SparseOperationKit: https://github.com/NVIDIA/HugeCTR/tree/master/sparse_operation_kit#documents

**Comment by JinrongGuo1** [comment]
I fixed it. It's because NCCL is not running properly in my docker environment, thus, multi GPUs cannot communicate as expected. By default, HugeCTR doesn't output NCCL log, I checked NCCL log by export "NCCL_DEBUG=INFO" and found the shared memory allocated for docker is too small. By recreating docker with larger "--shm-size", both NCCL and HugeCTR work well.

**Comment by bashimao** [comment]
Closing. Thanks @MuYu-zhi for your contribution! (see also [v3.4 release notes](https://github.com/NVIDIA-Merlin/HugeCTR/releases/tag/v3.4))
